py/torch_tensorrt/_enums.py

```diff
@@ -112,9 +113,11 @@ def _from(
                 return dtype.f32
             elif t == trt.bool:
                 return dtype.b
+            elif t == trt.bf16:
```

Comment:
https://github.com/pytorch/TensorRT/pull/2833#discussion_r1605149743

Reply:
I've addressed this now.